.github/workflows/deploy-staging.yml

```diff
@@ -1,7 +1,6 @@
 on:
-  push:
-    branches:
-      - main
+  release:
+    types: [published]
```

Comment:
this is in preparation for changing the release cadence.  stop pushing every PR to staging